cypress/integration/patientchart/chart.spec.js

```diff
@@ -0,0 +1,47 @@
+describe("Patient chart", function() {
```

Comment:
Remove the login portion of this test.
The remaining portion of this seems to test patient search. We should this into the esm-patient-home which presently houses the search functionality.

Reply:
One challenge I am still trying to fix is the authentication process. Cypress has a feature called commands which we can use to define the login process and invoke it before any test is run. I tried out a few things and will reorganize the tests accordingly once I manage to get the commands working